src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/ProblemMessageControllerTest.java

```diff
@@ -0,0 +1,182 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.controllers;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import io.restassured.response.Response;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.wacaseeventhandler.SpringBootFunctionalBaseTest;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.ccd.message.AdditionalData;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.ccd.message.EventInformation;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.ccd.message.EventInformationMetadata;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.ccd.message.EventInformationRequest;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.jobs.JobName;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.model.CaseEventMessage;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Collectors;
+
+import static net.serenitybdd.rest.SerenityRest.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static uk.gov.hmcts.reform.wacaseeventhandler.CreatorObjectMapper.asJsonString;
+
+@Slf4j
+public class ProblemMessageControllerTest extends SpringBootFunctionalBaseTest {
+    protected static final ObjectMapper OBJECT_MAPPER = new ObjectMapper()
+        .setPropertyNamingStrategy(PropertyNamingStrategies.UPPER_CAMEL_CASE)
+        .registerModule(new JavaTimeModule())
+        .registerModule(new Jdk8Module());
+
+
+    private LocalDateTime eventTimestamp1;
+    private LocalDateTime holdUntilTimestamp;
+
+    @Before
+    public void setup() {
+        eventTimestamp1 = LocalDateTime.parse("2020-03-27T12:56:10.403975").minusDays(1);
+        holdUntilTimestamp = LocalDateTime.parse("2020-03-27T12:56:10.403975").plusDays(10);
+    }
+
+    @Ignore("Ignore until AZURE_SERVICE_BUS_FEATURE_TOGGLE is turned on")
```

Comment:
can we verify this locally regardless of feature toggle? in an isolated downtime period perhaps?

Reply:
The test has been removed because it can not be tested in FT due to the workflow process.
the process will move the message quickly to process state, and the message will not be stuck in ready state long enough to test it.